uc-catalog-cloning/README.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "UC Catalog Cloning1"
```

Comment:
what is `Cloning1`? looks like a typo :) do you want it to be this short?

Reply:
Thanks for noticing it

Reply:
is it fixed?

Reply:
yes, it should be. I had committed it